tests/protocol/HttpRequestPredicates.py

```diff
@@ -11,6 +11,22 @@ class HttpRequestPredicates(object):
     def is_goal_state_request(url):
         return url.lower() == 'http://{0}/machine/?comp=goalstate'.format(restutil.KNOWN_WIRESERVER_IP)
 
+    @staticmethod
```

Comment:
I only needed is_certificates_request, but since I was here I added the rest as well